config.json

```diff
@@ -18,6 +18,17 @@
 	    ]
 	},
 	"propagatedmount": "/mnt",
+	"mounts": [
+		{
+		"name": "KeyPath",
+		"description": "Path to SSH keys",
+		"source": "/tmp",
+		"destination": "/tmp/sshfs",
```

Comment:
@stevvooe Only if this is run as a service in swarm mode - I think this will only be possible with swarm mode plugins, which hasn't been implemented yet.

Agree that secrets would be the best way to go, I'm just not aware that this is possible yet since AFAIK `docker plugin install` does not utilize swarm mode.